packages/flutter_markdown/test/text_test.dart

```diff
@@ -136,6 +136,26 @@ void defineTests() {
         expectTextStrings(widgets, <String>['line 1. line 2.']);
       },
     );
+
+    testWidgets(
+      'soft line break',
+          (WidgetTester tester) async {
+        const String data = 'line 1.\nline 2.';
+        await tester.pumpWidget(
+          boilerplate(
+            const MarkdownBody(
+              data: data,
+              softLineBreakPattern: true,
+            ),
+          ),
+        );
+
+        final Iterable<Widget> widgets = tester.allWidgets;
+        expectWidgetTypes(widgets,
+            <Type>[Directionality, MarkdownBody, Column, Wrap, RichText]);
+        expectTextStrings(widgets, <String>['line 1.\nline 2.']);
```

Comment:
What would the output be if `softLineBreakPattern` was set to false?

Reply:
Same style as current master, no change